### ISSUE TITLE
DB Error 1053

### ISSUE THREAD

**Issue by hellquist** [friendica/friendica#9997]
- [X] I have searched open and closed issues for duplicates

### Bug Description

I was asked to submit my database errors [here by Michael Vogel](https://pirati.ca/display/ec054ce7-7760-224c-d426-08d369931464?owt=2d4546f4deb077df2ad1afb9455448f6). I can't really explain them, or what goes wrong, but was told the errors themselves contain all the needed information, so here it goes:

```2021-03-07T01:00:40Z worker [ERROR]: DB Error {"code":1053,"error":"Server shutdown in progress","callstack":"Database::update, DatabaseCache::set, JsonLD::documentLoader, JsonLdProcessor::_retrieveContextUrls, JsonLdProcessor::expand, JsonLdProcessor::compact, jsonld_compact, JsonLD::compact","params":"SELECT * FROM `cache` WHERE (`k` = 'documentLoader:https:\/\/hub.netzgemeinde.eu\/apschema\/v1.9') LIMIT 1"} - {"file":"Database.php","line":662,"function":"p","uid":"475d0f","process_id":2912977}```

### Platform Info

Friendica Version:

Friendica Source: Friendica 'Red Hot Poker' 2021.03-dev - 1406

PHP version: 7.4.3

SQL version: 8.0.23

**Comment by MrPetovan** [comment]
Thank you for the report, see error message: `"Server shutdown in progress"`

This error is independent from Friendica. This query was interrupted because the database server itself was shutting down.

**Comment by hellquist** [comment]
> Thank you for the report, see error message: `"Server shutdown in progress"`
> 
> This error is independent from Friendica. This query was interrupted because the database server itself was shutting down.

Ah, haha, indeed. It was in the small hours of the morning, and I was just pleased I had managed to get everything back up and running after a ``git pull`` of the latest dev code. Before I did that ``git pull`` I had indeed stopped the database to reset my mysql root password that I apparently had set and forgotten, all to be able to do a db dump before running the update. :)

I should probably get myself some test/development servers, hehe (as opposed to be doing stuff on my production server) but as it is only an "instance-of-one" (for me) and even cheap servers cost *something*...:)

I shall read the log entries more closely going forward though, this one wouldn't be needed to clog up a bugs list.
